src/main/java/step3/controller/RacingGameController.java

```diff
@@ -44,23 +42,8 @@ public boolean hasNextRound() {
         return this.progressRound < this.gameRound;
     }
 
-    public RacingPosition getRacingPosition() {
+    public RacingCars getRacingCars() {
         return this.racingCars.stream()
```

Comment:
```suggestion
        return new RacingCars(racingCars);
```
stream 을 굳이 사용하지 않아도 될 것 같습니다!